Tasks/pigulevskaya/Task1/Новый текстовый документ.txt

```diff
@@ -0,0 +1,20 @@
+amount = int(input('Enter amount \n'))
+period = int(input('Enter number of years \n'))
+rate = float(input('Enter interest rate \n'))                      
```

Comment:
it's a money-related task, so I advise to use the Decimal data type for the calculation, -1 point